src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="post"
+      class="forms big__form"
```

Comment:
```suggestion
      class="form big__form"
```
it's one form, it can't be forms